diff --git a/src/app/App.tsx b/src/app/App.tsx
index 545b3d0..aedeffc 100644
--- a/src/app/App.tsx
+++ b/src/app/App.tsx
@@ -113,10 +113,10 @@ export default function App() {
           </Route>
 
           <Route path="/espace-consulaire/inscription" element={<Inscription />} />
-          <Route path="/espace-consulaire/verification-email" element={<VerificationEmail />} />
+          <Route path="/verifier-email" element={<VerificationEmail />} />
           <Route path="/espace-consulaire/login" element={<RessortissantLogin />} />
           <Route path="/espace-consulaire/mot-de-passe-oublie" element={<MotDePasseOublieRessortissant />} />
-          <Route path="/espace-consulaire/reinitialiser-mot-de-passe" element={<ReinitialiserMotDePasse />} />
+          <Route path="/reinitialiser-mot-de-passe" element={<ReinitialiserMotDePasse />} />
 
           <Route element={<RessortissantProtectedRoute />}>
             <Route path="/espace-consulaire" element={<RessortissantLayout />}>
